getting_started/create_vm/make_vm.sh

```diff
@@ -11,7 +11,7 @@ fi
 
 # Azure Resource Group
 RG=ccf-dev-rg
-# Region, only uksouth is supports DC*_v2 at the moment
+# Region, only uksouth supports DC*_v2 at the moment
```

Comment:
This isn't true any more, but I can't find a nice page with a list of supported regions either...